.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Description
+
+_Please include a summary of the changes, relevant motivation and context_.
+
+# Issue
+
+* _Redmine Ticket URL_ or _Github Issue ID_.
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code.
+- [ ] My code follows the style guidelines of this project.
+- [ ] I have made corresponding changes to the documentation.
+- [ ] New and existing tests pass locally with my changes.
+- [ ] This branch is up to date with the main branch.
```

Comment:
I think this is not needed. If Github allows merging without conflicts, then it should be fine. We can request updates on the branch if it is too outdated.